src/OpenSage.Game/Gui/Wnd/WndInputMessageHandler.cs

```diff
@@ -21,6 +24,20 @@ public WndInputMessageHandler(WndWindowManager windowManager, Game game)
 
         public override InputMessageResult HandleMessage(InputMessage message)
         {
+            bool GetControlAtPoint(in Point2D mousePosition, out Control control, out Point2D controlRelativePosition)
```

Comment:
My naming convention in other parts of the code is to use camelCase for local functions. Is PascalCase what ReSharper recommends?